ios/PolyPodApp/PolyPod/AppDelegate.swift

```diff
@@ -69,88 +84,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
     
-    // MARK: UISceneSession Lifecycle
+    // MARK: - UISceneSession Lifecycle
```

Comment:
Nit: I don't really know what this `MARK` stuff means, reminds me of Obj-C, but isn't `application` public, so should it not _not_ have a `-` here? (I thought that's what it means)

Reply:
This `-`, helps having the Marks nicely separated by a separator, like this:

![Screenshot 2022-01-13 at 17 31 59](https://user-images.githubusercontent.com/12482662/149359730-d3139219-fbc1-4ebf-b98b-02344dd9f0ef.png)